clients/TypeScript/package.json

```diff
@@ -18,12 +18,12 @@
     "cleanup": "yarn workspaces run cleanup && shx rm -rf node_modules",
     "generate-schema-types": "yarn workspace @cardano-ogmios/schema run generate-schema-types",
     "lint": "yarn workspaces run lint",
-    "mainnet:up": "docker-compose --project-directory ../.. -p ogmios-mainnet up",
+    "mainnet:up": "docker-compose -d --project-directory ../.. -p ogmios-mainnet up",
```

Comment:
Whoops :point_up: , `-d` option needs to be passed after `up` ... I fixed it for `testnet:up` but not for this one.